src/osd/ECBackend.cc

```diff
@@ -1642,11 +1846,15 @@ struct CallClientContexts :
 	   ++j) {
 	to_decode[j->first.shard].claim(j->second);
       }
-      ECUtil::decode(
+      int r = ECUtil::decode(
```

Comment:
At one point I added something that returned an error there.  I wanted to leave this in since ECUtil::decode() isn't void and I can handle errors now.